### ISSUE TITLE
Import

### ISSUE THREAD

**Issue by 4thcrack** [artisan-roaster-scope/artisan#110]
Hi, how do i import data from probat pilot shop?
i have the xml files.

is it possible to let artisan import the burner rate to?

**Comment by MAKOMO** [comment]
There is no import support for files from the Probat Pilot software yet. I add this on the (long) list of feature requests and keep this issue open for now.

**Comment by MAKOMO** [comment]
I just implemented Probat Pilot import, including the burner rate. Could you share one of your xml files to test this?

**Comment by 4thcrack** [comment]
[NeuesZip.zip](https://github.com/artisan-roaster-scope/artisan/files/661371/NeuesZip.zip)

Hi thanks for the answer.
i send you the file.

**Comment by MAKOMO** [comment]
Thanks for this. I discovered that I implemented the import/export only for recipes. Added now support for "histories" (import only as the Pilot does not has an import for histories, right?). Sadly the data in those exports has no decimal place and thus the result is not perfect and generates quite some noise on the RoR (a lot more than just recording directly with Artisan from a Probatone!). It is also obvious that something is wrong with the RoR computation in the Pilot software. Anyhow.

This feature will be released as part of Artisan v1.0. Expect more Probat Probatone support coming at the begin of next year;)

<img width="498" alt="screen shot 2016-12-19 at 21 35 16" src="https://cloud.githubusercontent.com/assets/13909011/21329003/c699411a-c636-11e6-8010-7bfbdbe03347.png">
<img width="1016" alt="screen shot 2016-12-19 at 21 54 39" src="https://cloud.githubusercontent.com/assets/13909011/21329006/c992a9f6-c636-11e6-9107-b2522dcf8e00.png">
<img width="735" alt="screen shot 2016-12-19 at 21 54 48" src="https://cloud.githubusercontent.com/assets/13909011/21329007/caa2d91a-c636-11e6-8b20-b8ada859cc78.png">
